components/org.wso2.carbon.identity.api.server.application.management/org.wso2.carbon.identity.api.server.application.management.v1/src/main/java/org/wso2/carbon/identity/api/server/application/management/v1/core/ServerApplicationMetadataService.java

```diff
@@ -64,6 +66,18 @@
 public class ServerApplicationMetadataService {
 
     private static final Log LOG = LogFactory.getLog(ServerApplicationMetadataService.class);
+    private static final String TOKEN_EP_SIGNATURE_ALGORITHMS_SUPPORTED = "OAuth.OpenIDConnect." +
+            "SupportedTokenEndpointSigningAlgorithms.SupportedTokenEndpointSigningAlgorithm";
+    private static final String ID_TOKEN_SIGNATURE_ALGORITHMS_SUPPORTED = "OAuth.OpenIDConnect." +
+            "SupportedIDTokenSigningAlgorithms.SupportedIDTokenSigningAlgorithm";
+    private static final String REQUEST_OBJECT_SIGNATURE_ALGORITHMS_SUPPORTED = "OAuth.OpenIDConnect." +
+            "SupportedRequestObjectSigningAlgorithms.SupportedRequestObjectSigningAlgorithm";
+    private static final String REQUEST_OBJECT_ENCRYPTION_ALGORITHMS_SUPPORTED = "OAuth.OpenIDConnect." +
+            "SupportedRequestObjectEncryptionAlgorithms.SupportedRequestObjectEncryptionAlgorithm";
+    private static final String REQUEST_OBJECT_ENCRYPTION_METHODS_SUPPORTED = "OAuth.OpenIDConnect." +
+            "SupportedRequestObjectEncryptionMethods.SupportedRequestObjectEncryptionMethod";
+    private static final String DEFAULT_SIGNATURE_ALGORITHM = "PS256";
```

Comment:
Get the server configured DEFAULT_SIGNATURE_ALGORITHM instead of hardcoding here.

Reply:
Fixed in 1f4de9bb1b4a721922a631b048ee16397e81baea